lang/src/org/partiql/lang/ast/ExprNodeToStatement.kt

```diff
@@ -335,3 +336,7 @@ private fun DataType.toAstType(): PartiqlAst.Type {
         }
     }
 }
+
+
+private fun SymbolicName.toPrimitive(): SymbolPrimitive =
+    SymbolPrimitive(this.name, this.metas.toIonElementMetaContainer())
```

Comment:
We had this exact function in the other PR, could we keep one implementation and reuse?

Reply:
Sorry for the confusion. The other PR (#298) needs to be merged first into the `visitor-transforms` feature branch.